src/cohort-builder.tsx

```diff
@@ -99,13 +88,21 @@ const CohortBuilder: React.FC = () => {
       });
       setPatients(rows);
       addToHistory(queryDescription, rows, searchParams.query);
-      setNotification({
+      showNotification({
+        title: t("success", "Success!"),
         kind: "success",
-        title: `Search is completed with ${rows.length} result(s)`,
+        critical: true,
+        description: `Search is completed with ${rows.length} result(s)`,
       });
       setIsLoading(false);
-    } catch (e) {
-      setNotification({ kind: "error", title: e.toString() });
+      setIsHistoryUpdated(true);
+    } catch (error) {
+      showNotification({
+        title: t("error", "Error"),
+        kind: "error",
+        critical: true,
+        description: error?.message,
```

Comment:
would `error.message` come with the selected translation? no, right?

Reply:
yeah